statistics/statistics.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="statistics.css" rel="stylesheet" />
+    <title>Statistics page</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header-img">
+            <img src="/images/corona3.png"  alt="Header corona image">
+        </div>
+        <div class="header-locatin ">
+            <img src="/images/loction.png" width="20px"  alt="">
+            <h1 class="country">
+                Palestine
+            </h1> 
+        </div>
```

Comment:
use semantic html `<fiqure>` instead of `<div>`
`<fiqcaption>` instead of `<h1>` and give it your style for font size and weight
And use `<section>` or `<article>` as containers instead of `<div>`
Reduce the use of `<div>` as you can
Check this link from your coursework [Sematic](https://github.com/gazaskygeeks/Coding-Foundations-course/blob/master/coursebook/Week%2001/session-02/semantic.md)
And read it carefully

Reply:
ok I will do that but
<fiqcaption> instead of <h1> ///
I mean h1 is a text not fiqcaption for figure